doc/getting-started/getting-started.rst

```diff
@@ -695,13 +695,13 @@ The ACRN hypervisor boots the Ubuntu Service VM automatically.
 
       [  0.000000] Hypervisor detected: ACRN
 
-#. Start the ServiceVM's system daemon for managing network configurations,
-   so the Device Model can create a bridge device that provides User VMs with
+#. Enable and start the Service VM's system daemon for managing network configurations,
+   so the Device Model can create a bridge device (acrn-br0) that provides User VMs with
    wired network access:
 
    .. code-block:: bash
 
-      systemctl enable --now systemd-networkd
+      sudo systemctl enable --now systemd-networkd
```

Comment:
That was not strictly required since the suer will be prompted to enter the privileged credentials anyway if he did not use sudo... not sure exactly how they do it but it's a nice feature. That said, there is certainly no harm in adding it here and it **does** require elevated privileges.

Thanks!